backend/pkg/server/handlers/infoClusters.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2025 ArubaKube S.r.l.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"log"
+
+	"github.com/gin-gonic/gin"
+	"github.com/liqotech/liqo/pkg/liqoctl/factory"
+
+	"github.com/ArubaKube/liqo-dashboard/pkg/utils/getters"
+)
+
+// GetV1Info implements the `GET /v1/info` route, returning a simple "ok" status to indicate that the server is running.
+func (s Server) GetV1Info(c *gin.Context) {
+	ctx := c.Request.Context()
+
+	f := factory.Factory{
+		KubeClient: s.nativeClient,
+		CRClient:   s.oClient,
```

Comment:
We are not passing the Liqo namespace here, which causes the status to be incorrectly determined. The health checker looks for pods in an unhealthy state in the Liqo namespace. However, since the namespace is an empty string, it ends up checking for unhealthy pods across the entire cluster.

I believe we should default to the Liqo namespace, while still allowing the option as a parameter in the server configuration (not forgetting to include this parameter in the values.yaml file of the Helm chart).